### PR TITLE
fix where in go clause

### PR DESCRIPTION
**Pull request by czpmango**
Fixes [https://github.com/vesoft-inc/nebula-graph/issues/761](https://github.com/vesoft-inc/nebula-graph/issues/761)

### CODE REVIEW

File: src/validator/GoValidator.cpp [hunk 1]
```diff
@@ -63,6 +63,7 @@ Status GoValidator::validateWhere(WhereClause* where) {
         ExpressionUtils::rewriteLabelAttribute<EdgePropertyExpression>(filter_);
     }
 
+    filter_ = where->filter();
     auto typeStatus = deduceExprType(filter_);
```
Comment (Shylock-Hg) [inline src/validator/GoValidator.cpp:67]: This don't report error when got nullptr ?

**Review by nevermore3 — APPROVED** [review]

File: src/validator/GoValidator.cpp [hunk 2]
```diff
@@ -63,6 +63,7 @@ Status GoValidator::validateWhere(WhereClause* where) {
         ExpressionUtils::rewriteLabelAttribute<EdgePropertyExpression>(filter_);
     }
 
+    filter_ = where->filter();
```
Comment (jievince) [inline src/validator/GoValidator.cpp:66]: filter_ = ExpressionUtils::rewriteLabelAttribute<EdgePropertyExpression>(laExpr));
where->setFilter(filter_)

File: src/validator/GoValidator.cpp (hunk 1 above)
Comment (czpmango) [reply src/validator/GoValidator.cpp:67]: It won't become nullptr when there is a where clause.

File: src/validator/GoValidator.cpp (hunk 2 above)
Comment (czpmango) [reply src/validator/GoValidator.cpp]: ACK.

**Review by jievince — APPROVED** [review]

**Review by nevermore3 — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**